### PR TITLE
016_steeringWheelLocation_RIGHT_response_transfered_fake_param_3_Apps_RAI_steeringWheelLocation

### PR DESCRIPTION
**Pull request by istoimenova**
Used common functions are implemented in
#603
Script is part of TS: #597

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

File: test_scripts/API/RAI/016_ATF_P_steeringWheelLocation_RIGHT_response_transfered_fake_param_3_Apps_RAI_steeringWheelLocation.lua [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+	self.mobileSession2:StartService(7)
+end
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_RAI_FirstApp_steeringWheelLocation()
+	local CorIdRegister = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+		
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.registerAppInterfaceParams.appName }})
+	EXPECT_RESPONSE(CorIdRegister, { success=true, resultCode = "SUCCESS", steeringWheelLocation = "RIGHT" })
```
Comment (OVikhrov) [inline test_scripts/API/RAI/016_ATF_P_steeringWheelLocation_RIGHT_response_transfered_fake_param_3_Apps_RAI_steeringWheelLocation.lua]: @istoimenova steeringWheelLocation is parameter of hmiCapabilities structure

File: test_scripts/API/RAI/016_ATF_P_steeringWheelLocation_RIGHT_response_transfered_fake_param_3_Apps_RAI_steeringWheelLocation.lua [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+	EXPECT_NOTIFICATION("OnHMIStatus", { systemContext = "MAIN", hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE"})
+end
+
+function Test:TestStep_RAI_SecondApp_steeringWheelLocation()
+	local CorIdRegister = self.mobileSession1:SendRPC("RegisterAppInterface", config.application2.registerAppInterfaceParams)
+		
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application2.registerAppInterfaceParams.appName }})
+	self.mobileSession1:ExpectResponse(CorIdRegister, { success=true, resultCode = "SUCCESS", steeringWheelLocation = "RIGHT" })
```
Comment (OVikhrov) [inline test_scripts/API/RAI/016_ATF_P_steeringWheelLocation_RIGHT_response_transfered_fake_param_3_Apps_RAI_steeringWheelLocation.lua]: @istoimenova steeringWheelLocation is parameter of hmiCapabilities structure

File: test_scripts/API/RAI/016_ATF_P_steeringWheelLocation_RIGHT_response_transfered_fake_param_3_Apps_RAI_steeringWheelLocation.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/API/RAI/016_ATF_P_steeringWheelLocation_RIGHT_response_transfered_fake_param_3_Apps_RAI_steeringWheelLocation.lua]: @OVikhrov, done in 7583c27327556f83482f4e8bd05e346f963b0fd5

File: test_scripts/API/RAI/016_ATF_P_steeringWheelLocation_RIGHT_response_transfered_fake_param_3_Apps_RAI_steeringWheelLocation.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/API/RAI/016_ATF_P_steeringWheelLocation_RIGHT_response_transfered_fake_param_3_Apps_RAI_steeringWheelLocation.lua]: @OVikhrov, done in 7583c27

**Pull request merged**